### DISCUSSION TITLE
Need help calling a method on a expanded q-table

### DISCUSSION THREAD

**Discussion by sideburn** [quasarframework/quasar#13121] [General - Components / Directives / etc] [answered]
Hello,

I have an expanding table taken from the example here:https://github.com/quasarframework/quasar/blob/dev/docs/public/examples/QTable/ExpandedRowInternal.vue

My problem is I am using this as an image thumb gallery and when a row is expanded, i need to make an axis call and fetch my data. I have all of that done I just don't know how to get my method to call and fetch the data when the row is expanded.

i have a method like this:    this.webServices.fetchImages(itemID);
and it populates the data that is being uses for my thumbnail tiles inside the expanded area:

`<div class="col-auto q-pa-xs" 
                v-for="item in rdWebServices.proofImages"
                :key="item.imageID"
              >
                <div class="image-container">
                  <q-img
                    :fit="imageFitMode"
                    :src="imageUrl(item)"
                    spinner-color="white"
                    style="width: 194px; height: 150px"
                  >
                  </q-img>
                </div>
              </div>`

Can someone help me out?

Thanks,

-Tavis

**Comment by metalsadman** [comment, marked as answer]
@sideburn do it in the click handle of your expanding component ie. `QBtn` https://codepen.io/metalsadman/pen/eYyrmag?editors=1010

**Reply by sideburn** [reply]
couldn't because it was already setting a pro. I put the entire thing in another component and used mounted

**Reply by MilosPaunovic** [reply]
If that's the case, you should provide minimal reproduction repo for someone to be able to play with it.
